runtime/onert/core/include/ir/operation/Loss.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_OPERATION_LOSS_H__
+#define __ONERT_IR_OPERATION_LOSS_H__
+
+#include "ir/Operation.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace operation
+{
+
+class Loss : public Operation
+{
+public:
+  enum Input
+  {
+    Y_PRED = 0,
+    Y_TRUE = 1
+    // TODO Add more inputs if necessary
+  };
+
+  enum class Type
+  {
+    CATEGORICAL_CROSSENTROPY
+  };
+
+  // NOTE It is not yet determined how to get the information of the previous activation.
+  //      If it is determined to get from the user, we have to consider whether to merge
+  //      this enum class and the `Type` enum class.
+  enum class PreActivationType
+  {
+    SOFTMAX
+  };
```

Comment:
@ragmani I don't understand why this is required. Could you please remove this and put the skeleton only in this PR? Let's review this in another PR.